src/Themes/Sample.Theme/Manifest.cs

```diff
@@ -7,7 +7,7 @@
     Version = "0.0.1",
     Website = "https://github.com/Lombiq/DotNest-Core-SDK",
     Description = "A sample DotNest theme for local development. It must be packaged and used as a Media Theme on DotNest.",
-    BaseTheme = "Lombiq.BaseTheme",
+    BaseTheme = "TheTheme",
```

Comment:
Why this change?

Reply:
First of all Lombiq.BaseTheme didn't add anything visible to the Sample.Theme. Secondly, the UI tests were failing because of some missing title or so after setup. But it is not failing now if I change back to the Lombiq.BaseTheme (This is weird). And finally, it just looks much much better, it is visible at first sight that we are using a base theme.

Reply:
OK then!